core/Hashing.h

```diff
@@ -12,19 +13,8 @@ inline unsigned int mix(unsigned int acc, unsigned int nw) {
 }
 
 inline unsigned int _hash(std::string_view utf8) {
-    // TODO: replace with http://www.sanmayce.com/Fastest_Hash/, see https://www.strchr.com/hash_functions
-    // and https://github.com/rurban/smhasher
-    auto end = utf8.end();
-    unsigned int res = 0;
-    auto it = utf8.begin();
-#ifndef HAS_SANITIZER
-#pragma clang loop unroll(enable)
-#endif
-    for (; it != end; it++) {
-        res = mix(res, *it - '!'); // "!" is the first printable letter in ASCII.
-        // This will help Latin1 but may harm utf8 multibyte
-    }
-    return res * HASH_MULT2 + static_cast<u4>(NameKind::UTF8);
+    // Take the lower 32-bits of the 64-bit hash. The 64-bit hash is faster than the 32-bit hash on 64-bit platforms.
```

Comment:
The author recommended this approach:
https://github.com/Cyan4973/xxHash/issues/453#issuecomment-696838445